roles/neutron-prerequisites/tasks/modules.yml

```diff
@@ -3,6 +3,7 @@
     state: present
     name: "{{ item }}"
   with_items: "{{ neutron_kernel_modules }}"
+  ignore_errors: true
```

Comment:
what's the reasoning for ignoring errors? If this fails, we may have lots more issues down the line.